guides/v2.2/extension-dev-guide/indexer-batch.md

```diff
@@ -130,7 +130,8 @@ Make sure that these indexers are in "Update By Schedule" mode. If "Update On Sa
 {: .bs-callout-info }
 The indexer table switching mechanism requires additional database storage.
 
-### Related topics
+{:.ref-header}
```

Comment:
Can you make class it one line instead of 2 here?
`{: .ref-header .no_toc}`

Reply:
Oh never mind, you don't need `.no_toc` here since it's not a heading, please remove it